modules/web/src/app/wizard/step/cluster/component.ts

```diff
@@ -172,76 +174,14 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
 
   // eslint-disable-next-line complexity
```

Comment:
We can remove this comment now because code has been moved inside `_initForm` method and we have a comment there.